### ISSUE TITLE
Support a11y needs of auro-combobox

### ISSUE THREAD

**Issue by jason-capsule42** [AlaskaAirlines/auro-menu#120]
## Is your feature request related to a problem? Please describe.

Auro-combobox needs additional tooling around how menu marks options as 'active'

## Describe the solution you'd like

The menu should not default to making the first menu option `active`. Doing so is causing a mismatch in the voiceover experience of auro-combobox. In stead we should activate the first available menuoption only on manual trigger.

Once a menuoption is made active, an event should be fired with the option attached in the details. This would be used by combobox to handle the voiceover experience as a user keyboard navigates the menu.
